content/blog/2018-07-26-educollab-community.md

```diff
@@ -0,0 +1,55 @@
+
+First, we need a community, a place to share information and build relationships. It is important to note that while we recognize (and envy!) The Carpentries and the strong [education community of practice](https://cookbook.carpentries.org) that has been developed around their workshops for teaching foundational coding and data science skills, we believe there is a place for a community of science educators who teach *with* R, but who may not identify as someone who teaches coding or programming skills. Currently, one of the main challenges for these educators is the availability and discoverability of course materials. There is an abundance of tutorials and a lack of curricular resources. For example, there are many individual tutorials, focused workshops or bootcamp materials, online courses, and lesson plan modules. All of these are great resources, and can be integrated within a long-form course curriculum. However, while it is helpful to leverage existing materials, adapting these materials in the classroom still requires preparation time. To use them successfully, educators usually need to think carefully about the dependencies of each individual resource (i.e., packages used, assumed prior R knowledge, assumed content domain knowledge), and how it fits into the larger temporal sequence of course materials and learning objectives. It also can be difficult to teach a class or lead a lab-based on others’ materials, written in someone else’s voice, and educators typically need to adapt the data or code to better meet their class’ needs or skill level. 
+
+On the other hand, there is a lack of integrated long-form  resources, where educators share full-length syllabi along with R-based learning materials. There are many reasons why these materials may not be open: lack of instructor confidence in sharing their teaching materials, lack of knowledge about tools and platforms for public sharing, and lack of support from institutions and departments. We would like to invite educators to begin to share their curricular materials, and offer a few recommended resources:
+
+- [Humanities Commons](https://hcommons.org)
+- [Open Science Framework](https://osf.io)
+- [GitHub repository](https://github.com) with recommended topic tags `teaching` and `rstats`. ([Sample search](https://github.com/search?utf8=%E2%9C%93&q=topic%3Ateaching+topic%3Arstats&ref=simplesearch))
+
+Even if they are made open, the problem remains of how to discover their availability; current methods tend to be time-dependent like sharing links in tweets or posts on community forums. If stored in a GitHub repository with the `teaching` and `rstats` topics added, one can search GitHub for these topics: currently there are 8 of them!
```

Comment:
I'm seeing 27 results at https://github.com/search?utf8=%E2%9C%93&q=topic%3Ateaching+topic%3Arstats&ref=simplesearch right now. In any case, I think that it'd be nice to include a screenshot like the one I attached

<img width="1008" alt="screen shot 2018-07-23 at 2 30 01 pm" src="https://user-images.githubusercontent.com/2288213/43095802-f9eb3e0c-8e84-11e8-9ed1-2758ecba837b.png">

Reply:
Thanks. I've updated the number of search results.

I do not want to include a screenshot, because it is too much emphasis and distracting from the body of the post.